specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -1157,6 +1136,32 @@
         }
       }
     },
+    "TargetPortalGroupUpdate": {
+      "type": "object",
+      "description": "iSCSI target portal group",
+      "required": [
+        "luns",
+        "acls"
+      ],
+      "properties": {
+        "luns": {
+          "type": "array",
+          "description": "Lun list to be exposed through the iSCSI target. Required",
+          "items": {
+            "$ref": "#/definitions/IscsiLun"
+          },
+          "minLength": 1,
+          "maxLength": 8
+        },
+        "acls": {
+          "type": "array",
+          "description": "Access Control List (ACL) for an iSCSI target lun",
+          "items": {
+            "$ref": "#/definitions/Acl"
```

Comment:
Please make sure all the 4 properties in ACL can be update： 

- initiatorIqn
- mappedLuns
- credentials.username
- credentials.password
If so, please also confirm, user name and password must update together.